md/assets.md

```diff
@@ -72,9 +72,14 @@ A page can not have more than one localization asset. When you upload a new `loc
 
 The list contains both page assets and the assets of the custom widgets used in this page.
 
-In the list, you can delete page assets only, view local assets, and edit external assets of a page. You can not manually change the order of assets in the page. You can also indicate whether an asset is active. To avoid dependency clashes with multiple version of assets, make sure only one version of an asset is active in a page.
+In the list, you can delete page assets only, view local assets, and edit external assets of a page. You can also indicate whether an asset is active. To avoid dependency clashes with multiple version of assets, make sure only one version of an asset is active in a page.
```

Comment:
Can we edit an external asset? 
If I am not wrong, we can edit the url if it is external, edit the content if it is internal